test/src/test/java/io/pravega/schemaregistry/integrationtest/TestEndToEnd.java

```diff
@@ -264,5 +261,10 @@ public void testLargeSchemas() {
     }
 
     abstract SchemaStore getStore();
+    
+    protected SchemaRegistryClient getClient() {
```

Comment:
nit: a method name prefixed with `get` is usually expected to simply return a ready value (like a getter), as opposed to computing the value on the value. So, a different name like `client()` or `newClient` for the method'd be more appropriate. 
That being said, it's not a serious problem here and it's OK if you want to keep the name `get..`.